src/main/java/DukeController.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+
+public class DukeController {
+
+    public static final int EVENT_STRING_LENGTH = 5;
+    public static final int DEADLINE_STRING_LENGTH = 8;
+    public static final int TODO_STRING_LENGTH = 4;
+    public static Task[] tasks = new Task[100];
+    public static int taskCount = 0;
+
+    public static void run(Scanner scanner) {
```

Comment:
Maybe can rename the function so that it gives more insight to what the function does